src/FSharpVSPowerTools.Core/Utils.fs

```diff
@@ -420,7 +420,32 @@ module String =
         elif value.Contains(pattern) then
             Some value
         else None
-     
+    
+    open System.IO
+
+    let getLines (str: string) =
+        use reader = new StringReader(str)
+        [|
+        let line = ref (reader.ReadLine())
+        while (!line |> function | null -> false | _ -> true) do
+            yield !line
+            line := reader.ReadLine()
+        if str.EndsWith("\n") then
+            // last trailing space not returned
+            // http://stackoverflow.com/questions/19365404/stringreader-omits-trailing-linebreak
+            yield String.Empty
+        |]
+
+    let getNonEmptyLines (str: string) =
```

Comment:
What do you think of introducing a private function that extract commonalities of these two functions using a boolean flag? It will be better in keeping two similar implementations in sync.